index.js

```diff
@@ -24,18 +23,31 @@ module.exports = {
     'max-params': ['warn', 2],
     'no-debugger': 'warn',
     'no-else-return': 'off',
-    'no-extra-semi': 'error',
     'no-restricted-imports': ['error', { patterns: ['../*'] }],
     'no-restricted-modules': ['error', { patterns: ['../*'] }],
     'prettier/prettier': 'error',
-    'quote-props': ['error', 'consistent-as-needed'],
     'react/destructuring-assignment': 'warn',
     'react/jsx-curly-brace-presence': 'off',
-    'react/jsx-filename-extension': ['error', { extensions: ['.js', '.jsx'] }],
+    'react/jsx-filename-extension': [
+      'error',
+      { extensions: ['.js', '.jsx', '.ts', '.tsx'] }
+    ],
     'react/jsx-one-expression-per-line': 'off',
     'react/jsx-wrap-multilines': 'off',
     'react/prefer-stateless-function': 'off',
-    'react/sort-comp': ['error', { order: ['static-methods', 'instance-variables', 'lifecycle', 'everything-else', 'render'] }],
+    'react/sort-comp': [
+      'error',
+      {
+        order: [
+          'static-variables',
+          'instance-variables',
+          'static-methods',
```

Comment:
maybe to hoist statics first?

Reply:
Do you want static methods before static variables? I’d prefer having variables first as they are usually shorter.

Reply:
Hmm, I think you make a good point! All good, ignore me :D